packages/react-pdf-toolkit/src/utils/is-link.ts

```diff
@@ -0,0 +1,9 @@
+export const isLink = (value: string): boolean => {
+  try {
+    new URL(value);
```

Comment:
How does this check compare to using something like zod's URL validation?

Reply:
P0

Reply:
![image](https://github.com/ballerine-io/ballerine/assets/23522869/1015c118-30ee-456f-ae9f-4e7f09bd58c9)
Just validating if URL valid here. No need to add package for single usage.